Header header


bool has_new_local_view_cell_

uint32 most_active_lv_cell_
uint32[] pose_cell_index

geometry_msgs/Point traveled_distance_
geometry_msgs/Point ground_truth_

PlaceCellActivity pc_activity_
LocalViewCell[]  lv_cells_active_



